crates/runtime/src/accelerated_table.rs

```diff
@@ -68,69 +68,69 @@ mod synchronized_table;
 
 #[derive(Debug, Snafu)]
 pub enum Error {
-    #[snafu(display("Unable to get data from connector: {source}"))]
+    #[snafu(display("Failed to get data from the connector.\n{source}"))]
     UnableToGetDataFromConnector { source: DataFusionError },
 
-    #[snafu(display("Dataset refresh failed with error: {source}"))]
+    #[snafu(display("Failed to refresh the dataset.\n{source}"))]
     FailedToRefreshDataset { source: DataFusionError },
 
-    #[snafu(display("Unable to scan table provider: {source}"))]
+    #[snafu(display("Failed to read from the dataset.\n{source}"))]
     UnableToScanTableProvider {
         source: datafusion::error::DataFusionError,
     },
 
-    #[snafu(display("Unable to create MemTable from data update: {source}"))]
+    #[snafu(display("Failed to create an in-memory table from a data update.\n{source}"))]
```

Comment:
This feels like an internal error? Why does the user care about in-memory tables and data updates?